brainstormforce/astra
# AST-2063 - view cart link added back when ajax add to cart enabled

### Description

- Previously there was a view cart link which would get trigged on ajax add to cart. This was removed in the Woocomerce update due to ui reasons. Added it back.

### Screenshots
https://d.pr/i/a3Ab9j

### Types of changes



-  View Cart link is visible and working as expected when AJAX Add to Cart is enabled.




### How has this been tested?


- Check if the view cart link works as intended.
- Check if the view cart link shows up on frontend.
- View button link should only show up for ajax add to cart is enabled.
- If Off canvas cart is enabled then we should not show view cart link button.

### Checklist:
